ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -133,7 +126,10 @@ type EraCrypto era = Crypto era
 -- needed to determine the hard fork point. In the future this should be
 -- replaced with an appropriate API - see
 -- https://github.com/input-output-hk/ouroboros-network/issues/2890
-class ( SL.ShelleyBasedEra era
+class ( Core.EraSegWits era
```

Comment:
The Haddock on this class is now out of date; eg it mentions `SL.ShelleyBasedEra`. Do you feel able to update it?

Reply:
I've updated the haddock.